google-cloud-logging/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+### 1.6.0 / 2019-01-22
+
+- Default `::Logging::Entry#insert_id` to pseudo-random value (#2799)
```

Comment:
I do not like listing pull requests in the changelog.

We have used `*` for bullet lists before this entry, why the change?

Reply:
When you say `::Logging` you refer to the `Logging` in the top-level namespace, not `Google::Cloud::Logging`.